assets/src/block-editor/components/with-media-library-notice.js

```diff
@@ -57,24 +57,57 @@ export default ( InitialMediaUpload, minImageDimensions ) => {
 		 */
 		initFeaturedImage() {
 			const FeaturedImageSelectMediaFrame = getSelectMediaFrame( FeaturedImageToolbarSelect );
+			const wpLibrary = wp.media.controller.Library;
+
+			const Library = wpLibrary.extend( {
+				defaults: {
+					...wpLibrary.prototype.defaults,
+					title: __( 'Choose image', 'amp' ),
+					library: wp.media.query( { type: 'image' } ),
+					multiple: false,
+					date: false,
+					priority: 20,
+					// Note: These suggestions are shown in the media library image browser.
+					suggestedWidth: EXPECTED_WIDTH,
+					suggestedHeight: EXPECTED_HEIGHT,
+				},
+
+				activate() {
+					this.updateSelection();
+					this.frame.on( 'open', this.updateSelection, this );
+
+					// eslint-disable-next-line prefer-rest-params
+					wpLibrary.prototype.activate.apply( this, arguments );
+				},
+
+				deactivate() {
+					this.frame.off( 'open', this.updateSelection, this );
+
+					// eslint-disable-next-line prefer-rest-params
+					wpLibrary.prototype.deactivate.apply( this, arguments );
+				},
+
+				updateSelection() {
+					const selection = this.get('selection'),
+						id = wp.media.view.settings.post.featuredImageId;
+					let attachment;
+
+					if ( '' !== id && -1 !== id ) {
```

Comment:
If the post type doesn't support featured images, `featuredImageId` will be undefined here:

![image](https://user-images.githubusercontent.com/2676022/77770859-88c84980-700b-11ea-8476-5d2566480697.png)

even if this code shouldn't load in that case, this should include a falsey check as a sanity check.

Reply:
Thanks! Made suggested changes in https://github.com/ampproject/amp-wp/pull/4453/commits/7cfd2d6e8799011065c29fd6a14b61137ca60e43.